cmd/temporalite/main_test.go

```diff
@@ -0,0 +1,45 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the MIT License.
```

Comment:
Running `go run ./internal/copyright` puts a header that doesn't validate on `--verify-only`, but we can fix that later

Reply:
Well, there's the problem 🤦 https://github.com/temporalio/temporalite/blob/f3d60547a26df3fcde912d22d56f8aaedbecff69/internal/copyright/main.go#L171

I can try whipping up a quick fix so we can merge with the right copyright headers.

Reply:
Just kidding that's always been a useless function. But it's a simple fix: https://github.com/temporalio/temporalite/pull/137

Reply:
Please go ahead and resolve this thread after updating the header now that the fix is merged.